src/Juvix/Compiler/Backend/Html/Translation/FromTyped.hs

```diff
@@ -288,7 +288,7 @@ goTopModule cs m = do
   htmlOpts <- ask @HtmlOptions
   runReader (htmlOpts {_htmlOptionsKind = HtmlDoc}) $ do
     fpath <- moduleDocPath m
-    Prelude.embed (putStrLn ("Writing " <> pack (toFilePath fpath)))
+    putStrLn ("Writing " <> pack (toFilePath fpath))
```

Comment:
Same with removing `embed`

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582